GUI/community-version.py

```diff
@@ -262,14 +266,15 @@ def handle_image(image_file_path):
             else:
                 raise ms.showerror("The file extension did not match as txt file!")
         except Exception as e:
-            ms.showerror("\33[101mOops, I think you have choosed wrong file extension. Please give a svg file name e.g., output.txt \033[0m")
+            ms.showerror("\33[101mOops, I think you have chosen an incorrect file extension. Hint: Please enter name of SVG. e.g., output.txt \033[0m")
     return capture
 
 def main():
     global entry1
     image_file_path = entry1.get()
     ascii_img = handle_image(image_file_path)
-    print(ascii_img)
+    # print(ascii_img)
+    print(pyfiglet.figlet_format(ascii_img))
```

Comment:
This change breaks the ASCII art because `pyfiglet` is for formatting actual text, not random ASCII characters that make an image.